src/pybind/mgr/dashboard/frontend/src/app/shared/api/user.service.ts

```diff
@@ -53,14 +53,10 @@ export class UserService {
   }
 
   validatePassword(password: string, username: string = null, oldPassword: string = null) {
-    let params = new HttpParams();
-    params = params.append('password', password);
-    if (username) {
-      params = params.append('username', username);
-    }
-    if (oldPassword) {
-      params = params.append('old_password', oldPassword);
-    }
-    return this.http.post('api/user/validate_password', null, { params });
+    return this.http.post('api/user/validate_password', {
+      password: password,
+      username: username,
+      old_password: oldPassword
+    });
```

Comment:
It's funny... so our back-end takes the params from wherever they were defined? 😲

Reply:
It looks like it is. I went for this change because earlier I did a same change in a different part of the code.